
{'date': 'Thu Jul 09 2020 17:01:53.9 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 12
  """" Mostra a cena da praia """
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Jul 09 2020 17:02:40.144 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 15
    calcada().vai()
NameError: name 'calcada' is not defined
'''},
{'date': 'Thu Jul 09 2020 17:03:18.683 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 13
    calcada().vai()
NameError: name 'calcada' is not defined
'''},
{'date': 'Thu Jul 09 2020 17:03:38.356 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 13
    Calcada().vai()
  module <module> line 12
    		cena(cena_praia).vai()
NameError: name 'cena' is not defined
'''},
{'date': 'Thu Jul 09 2020 17:11:15.834 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 12
  """" Mostra a cena da praia """
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Jul 09 2020 17:14:32.321 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 12
  """" Mostra a cena da praia """
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Jul 09 2020 17:14:50.256 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 12
  """" Mostra a cena da praia """
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Jul 09 2020 17:15:28.421 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 12
  """" Mostra a cena da praia """
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Jul 09 2020 17:19:32.172 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 15
    if name== __main__:
NameError: name 'name' is not defined
'''},
{'date': 'Thu Jul 09 2020 17:30:21.850 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 15
  self.banhista = Elemento(BANHISTA, x=100, y=200cena=self.cena)
                                                  ^
SyntaxError: invalid syntax
'''},
{'date': 'Sun Jul 12 2020 15:35:09.208 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Hello Monica
Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    	f.write(name)
AttributeError: '_io.TextIOWrapper' object has no attribute 'write'
'''},
{'date': 'Sun Jul 12 2020 16:27:38.127 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 38
    circus(2, MASMORRA)
TypeError: 'module' object is not callable
'''},
{'date': 'Sun Jul 12 2020 16:29:27.74 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 38
    circus(2, MASMORRA)
TypeError: 'module' object is not callable
'''},
{'date': 'Sun Jul 12 2020 16:30:48.366 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 38
    self.circus = circus(2, MASMORRA)
TypeError: 'module' object is not callable
'''},
{'date': 'Sun Jul 12 2020 16:34:07.271 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 37
    class circo:
  module <module> line 43
    	circo().vai()
NameError: name 'circo' is not defined
'''},
{'date': 'Tue Jul 14 2020 17:15:13.173 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 4
    from _spy.vitollino.main import Cena, Elemento, style
ImportError: cannot import name 'style'

ImportError
Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 264
    action()
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 4
    from _spy.vitollino.main import Cena, Elemento, style
'''},
{'date': 'Tue Jul 14 2020 17:18:06.830 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 67
    Eventos().vai()
  module <module> line 23
    document.bind("keydown", self.anda_banhista)  # captura o evento de teclado
AttributeError: 'Eventos' object has no attribute 'anda_banhista'
'''},
{'date': 'Tue Jul 14 2020 17:18:07.672 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 67
    Eventos().vai()
  module <module> line 23
    document.bind("keydown", self.anda_banhista)  # captura o evento de teclado
AttributeError: 'Eventos' object has no attribute 'anda_banhista'
'''},
{'date': 'Tue Jul 14 2020 17:25:12.175 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 58
  if boneco.x = 500 and boneco.y = 500:
               ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Jul 14 2020 17:25:48.815 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 58
  if self.boneco.x = 500 and boneco.y = 500:
                    ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Jul 14 2020 17:38:36.433 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 53
  if self.boneco.x = 500 and self.boneco.y = 500:
                    ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Jul 29 2020 17:18:42.537 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 22
    historia()
  module <module> line 19
    cena_historia = cena(link_balao)
NameError: name 'cena' is not defined
'''},
{'date': 'Wed Jul 29 2020 17:19:34.573 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 22
    historia()
  module <module> line 19
    cena_historia = cena(link_balao)
NameError: name 'cena' is not defined
'''},
{'date': 'Wed Jul 29 2020 17:20:58.769 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 19
  cena_historia = cena(""https://www.google.com/imgres?imgurl=https%3A%2F%2Fstatic.vecteezy.com%2Fsystem%2Fresources%2Fthumbnails%2F000%2F235%2F346%2Foriginal%2Fspeech-bubble-pop-art-comic-book-background.jpg&imgrefurl=https%3A%2F%2Fpt.vecteezy.com%2Farte-vetorial%2F235346-fundo-de-quadrinhos-do-discurso-bolha-pop-art&tbnid=a4iHDHXihmIeBM&vet=12ahUKEwiEpeXhovPqAhWDCLkGHTYCCdYQMygcegUIARDPAQ..i&docid=d0bWFVVCq0leWM&w=3601&h=3601&q=historia%20em%20quadrinhos%20vazia%20desenho&safe=active&ved=2ahUKEwiEpeXhovPqAhWDCLkGHTYCCdYQMygcegUIARDPAQ")
                          ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Jul 29 2020 17:21:25.923 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 22
    historia()
  module <module> line 19
    cena_historia = cena("https://www.google.com/imgres?imgurl=https%3A%2F%2Fstatic.vecteezy.com%2Fsystem%2Fresources%2Fthumbnails%2F000%2F235%2F346%2Foriginal%2Fspeech-bubble-pop-art-comic-book-background.jpg&imgrefurl=https%3A%2F%2Fpt.vecteezy.com%2Farte-vetorial%2F235346-fundo-de-quadrinhos-do-discurso-bolha-pop-art&tbnid=a4iHDHXihmIeBM&vet=12ahUKEwiEpeXhovPqAhWDCLkGHTYCCdYQMygcegUIARDPAQ..i&docid=d0bWFVVCq0leWM&w=3601&h=3601&q=historia%20em%20quadrinhos%20vazia%20desenho&safe=active&ved=2ahUKEwiEpeXhovPqAhWDCLkGHTYCCdYQMygcegUIARDPAQ")
NameError: name 'cena' is not defined
'''},
{'date': 'Wed Jul 29 2020 17:30:05.42 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 53
  if self.boneco.x = 500 and self.boneco.y = 500:
                    ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Jul 29 2020 17:32:05.242 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 26
    self.vai()
AttributeError: 'GUI' object has no attribute 'vai'
'''},
{'date': 'Wed Jul 29 2020 17:38:57.214 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 27
    Eventos().vai()
NameError: name 'Eventos' is not defined
'''},
{'date': 'Wed Jul 29 2020 17:39:49.676 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 28
    vai()
TypeError: vai() missing 1 positional argument: self
'''},
{'date': 'Wed Jul 29 2020 17:40:42.781 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 28
    vai(self)
  module <module> line 24
    self.ambiente.vai()
AttributeError: 'GUI' object has no attribute 'ambiente'
'''},
{'date': 'Wed Jul 29 2020 17:41:16.715 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 29
    vai(self)
  module <module> line 24
    self.ambiente = Cena(img = self.link_balao)
AttributeError: 'GUI' object has no attribute 'link_balao'
'''},
{'date': 'Wed Jul 29 2020 17:43:24.253 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 29
    vai(self)
  module <module> line 24
    self.ambiente = Cena(self.link_balao)
AttributeError: 'GUI' object has no attribute 'link_balao'
'''},
{'date': 'Wed Jul 29 2020 17:44:06.453 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 29
    vai(self)
  module <module> line 24
    self.ambiente = Cena(self.link_balao2)
AttributeError: 'GUI' object has no attribute 'link_balao2'
'''},
{'date': 'Wed Jul 29 2020 17:46:09.980 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 23
    vai()
TypeError: vai() missing 1 positional argument: self
'''},
{'date': 'Thu Jul 30 2020 14:59:09.244 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 42
  self.v = vitollino()
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Jul 30 2020 15:08:08.699 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 14
    class Kwarwp():
  module <module> line 39
    def __init__(self, vitollino=None, mapa=MAPA_INICIO, medidas={}):
NameError: name 'MAPA_INICIO' is not defined
'''},
{'date': 'Thu Jul 30 2020 15:08:48.197 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 14
    class Kwarwp():
  module <module> line 39
    def __init__(self, vitollino=None, mapa=MAPA_INICIO, medidas={}):
NameError: name 'MAPA_INICIO' is not defined
'''},
{'date': 'Thu Jul 30 2020 15:12:21.46 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 14
    class Kwarwp():
  module <module> line 43
    Jogo.c
NameError: name 'Jogo' is not defined
'''},
{'date': 'Thu Jul 30 2020 15:12:42.499 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 43
    Jogo.c
NameError: name 'Jogo' is not defined
'''},
{'date': 'Thu Jul 30 2020 15:17:02.640 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 42
    cria(self)
NameError: name 'cria' is not defined
'''},
{'date': 'Thu Jul 30 2020 15:17:53.530 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 42
    kwarwp.cria(self)
NameError: name 'kwarwp' is not defined
'''},
{'date': 'Thu Jul 30 2020 15:18:15.390 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 42
    kwarwp()
NameError: name 'kwarwp' is not defined
'''},
{'date': 'Thu Jul 30 2020 15:18:24.758 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 42
    kwarwp().cria()
NameError: name 'kwarwp' is not defined
'''},
{'date': 'Thu Jul 30 2020 15:18:54.615 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 42
    Kwarwp().cria()
  module <module> line 27
    self.v = vitollino()
TypeError: 'NoneType' object is not callable
'''},
{'date': 'Thu Jul 30 2020 15:37:48.112 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 43
    Kwarwp().cria()
  module <module> line 27
    self.v = vitollino()
TypeError: 'NoneType' object is not callable
'''},
{'date': 'Thu Jul 30 2020 15:38:28.550 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 45
    Kwarwp().cria()
  module <module> line 29
    self.v = vitollino()
TypeError: 'NoneType' object is not callable
'''},
{'date': 'Thu Jul 30 2020 15:40:07.443 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 45
    Kwarwp().cria()
  module <module> line 29
    self.v = vitollino()
TypeError: 'NoneType' object is not callable
'''},
{'date': 'Thu Jul 30 2020 15:40:36.942 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 45
    Kwarwp().cria()
  module <module> line 29
    self.v = vitollino()
TypeError: 'NoneType' object is not callable
'''},
{'date': 'Thu Jul 30 2020 15:41:21.897 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 45
    Kwarwp()
  module <module> line 29
    self.v = vitollino()
TypeError: 'NoneType' object is not callable
'''},
{'date': 'Thu Jul 30 2020 15:41:50.289 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 45
    Kwarwp()
  module <module> line 29
    self.v = vitollino()
TypeError: 'NoneType' object is not callable
'''},
{'date': 'Thu Jul 30 2020 15:52:19.151 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 45
    Kwarwp()
  module <module> line 28
    self.v = vitollino()
TypeError: 'NoneType' object is not callable
'''},
{'date': 'Thu Jul 30 2020 16:10:26.170 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 48
    Kwarwp().cria()
  module <module> line 31
    self.v = vitollino()
TypeError: 'NoneType' object is not callable
'''},
{'date': 'Thu Jul 30 2020 16:10:47.418 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 48
    Kwarwp().cria()
  module <module> line 36
    cena = self.v.c(self.SOLO)
AttributeError: 'Kwarwp' object has no attribute 'v'
'''},
{'date': 'Thu Jul 30 2020 16:13:05.764 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 14
    from _spy.vitollino.main import Jogo.a, Jogo.c
ImportError: cannot import name 'a'

ImportError
Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 264
    action()
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 14
    from _spy.vitollino.main import Jogo.a, Jogo.c
'''},
{'date': 'Thu Jul 30 2020 16:13:46.140 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 48
    Kwarwp().cria()
  module <module> line 36
    cena = self.v.c(self.SOLO)
AttributeError: 'Kwarwp' object has no attribute 'v'
'''},
{'date': 'Thu Jul 30 2020 16:14:04.602 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 48
    Kwarwp().cria()
  module <module> line 31
    self.v = vitollino()
TypeError: 'NoneType' object is not callable
'''},
{'date': 'Thu Jul 30 2020 16:15:40.797 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 49
    Kwarwp().cria()
  module <module> line 30
    self.v = vitollino()
TypeError: 'NoneType' object is not callable
'''},
{'date': 'Thu Jul 30 2020 16:18:00.99 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 47
    Kwarwp().cria()
  module <module> line 28
    self.v = vitollino()
TypeError: 'NoneType' object is not callable
'''},
{'date': 'Thu Jul 30 2020 16:19:06.897 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 49
    Kwarwp().cria()
  module <module> line 30
    self.v = vitollino()
TypeError: 'NoneType' object is not callable
'''},
{'date': 'Thu Jul 30 2020 16:25:59.222 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 52
  STYLE["heigth"] = 600px
                        ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Jul 30 2020 16:41:39.164 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 71
    Kwarwp(Jogo)
  module <module> line 53
    self.cena = self.cria(mapa=mapa) if vitollino else None
  module <module> line 62
    [self.cria_elemento( x=i*lado, y=j*lado+lado, cena=cena)
AttributeError: 'Kwarwp' object has no attribute 'cria_elemento'
'''},
{'date': 'Thu Jul 30 2020 16:43:53.271 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 75
    Kwarwp(Jogo)
  module <module> line 53
    self.cena = self.cria(mapa=mapa) if vitollino else None
  module <module> line 62
    [self.cria_elemento( x=i*lado, y=j*lado+lado, cena=cena)
  module <module> line 69
    return self.v.a(self.GLIFOS[imagem], w=lado, h=lado, x=i*lado, y=j*lado+lado, cena=cena)
KeyError: +
'''},
{'date': 'Thu Jul 30 2020 16:44:31.302 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 73
    Kwarwp(Jogo)
  module <module> line 51
    self.cena = self.cria(mapa=mapa) if vitollino else None
  module <module> line 60
    [self.cria_elemento( x=i*lado, y=j*lado+lado, cena=cena)
  module <module> line 67
    return self.v.a(self.GLIFOS[imagem], w=lado, h=lado, x=i*lado, y=j*lado+lado, cena=cena)
KeyError: +
'''},
{'date': 'Thu Jul 30 2020 16:46:08.685 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 73
    Kwarwp(Jogo)
  module <module> line 51
    self.cena = self.cria(mapa=mapa) if vitollino else None
  module <module> line 60
    [self.cria_elemento( x=i*lado, y=j*lado+lado, cena=cena)
  module <module> line 67
    return self.v.a(self.GLIFOS[imagem], w=lado, h=lado, x=i*lado, y=j*lado+lado, cena=cena)
KeyError: +
'''},
{'date': 'Wed Aug 05 2020 14:39:10.540 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 113
    Kwarwp(Jogo, mapa=MAPA_ROCHA)
  module <module> line 62
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 70
    fabrica = {
  module <module> line 1
    (IMGUR)
NameError: name 'IMGUR' is not defined
'''},
{'date': 'Wed Aug 05 2020 14:39:52.599 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 113
    Kwarwp(Jogo, mapa=MAPA_ROCHA)
  module <module> line 62
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 70
    fabrica = {
  module <module> line 1
    (IMGUR)
NameError: name 'IMGUR' is not defined
'''},
{'date': 'Wed Aug 05 2020 14:40:28.448 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 113
    Kwarwp(Jogo, mapa=MAPA_ROCHA)
  module <module> line 62
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 70
    fabrica = {
  module <module> line 1
    (IMGUR)
NameError: name 'IMGUR' is not defined
'''},
{'date': 'Wed Aug 05 2020 14:48:35.193 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 115
    Kwarwp(Jogo, mapa=MAPA_ROCHA)
  module <module> line 62
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 72
    fabrica = {
AttributeError: 'Kwarwp' object has no attribute 'vazio'
'''},
{'date': 'Wed Aug 05 2020 14:49:26.661 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 115
    Kwarwp(Jogo, mapa=MAPA_ROCHA)
  module <module> line 62
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 72
    fabrica = {
AttributeError: 'Kwarwp' object has no attribute 'vazio'
'''},
{'date': 'Wed Aug 05 2020 14:52:07.810 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 115
    Kwarwp(Jogo, mapa=MAPA_ROCHA)
  module <module> line 62
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 92
    self.taba = {(i, j): fabrica[imagem].objeto(
KeyError: +
'''},
{'date': 'Wed Aug 05 2020 15:14:13.444 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 115
    Kwarwp(Jogo, mapa=MAPA_ROCHA)
  module <module> line 62
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 92
    self.taba = {(i, j): fabrica[imagem].objeto(
KeyError: %
'''},
{'date': 'Wed Aug 05 2020 16:27:22.106 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 109
  Kwarwp(Jogo, mapa=MAPA_ROCHA, {w=700, h=600})
                                  ^
SyntaxError: can't assign to function call 
'''},
{'date': 'Wed Aug 05 2020 16:28:45.493 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 109
  Kwarwp(Jogo, mapa=MAPA_ROCHA, {w=700, h=600})
                                  ^
SyntaxError: can't assign to function call 
'''},
{'date': 'Wed Aug 05 2020 16:29:26.469 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 109
  Kwarwp(Jogo, mapa=MAPA_ROCHA, {w:700, h:600})
                                              ^
SyntaxError: non-keyword arg after keyword arg
'''},
{'date': 'Wed Aug 05 2020 16:29:54.932 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 109
  Kwarwp(Jogo, mapa=MAPA_ROCHA, {"w":700, "h":600})
                                                  ^
SyntaxError: non-keyword arg after keyword arg
'''},
{'date': 'Thu Aug 06 2020 14:47:49.697 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 283
  Kwarwp(Jogo, mapa=MAPA_INICIO)
                                ^
SyntaxError: decorator expects function
'''},
{'date': 'Thu Aug 06 2020 14:49:28.398 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 283
  Kwarwp(Jogo)
              ^
SyntaxError: decorator expects function
'''},
{'date': 'Thu Aug 06 2020 14:51:16.634 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 283
  Kwarwp(Jogo, mapa=MAPA_INICIO)
                                ^
SyntaxError: decorator expects function
'''},
{'date': 'Thu Aug 06 2020 14:53:36.815 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 282
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 190
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 221
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 240
    coisa = Indio(imagem, x=0, y=0, cena=cena, taba=self)
TypeError: __init__() got an unexpected keyword argument 'taba'
'''},
{'date': 'Thu Aug 06 2020 14:55:28.464 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 282
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 190
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 221
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 240
    coisa = Indio(imagem, x=0, y=0, cena=cena, taba=self)
TypeError: __init__() got an unexpected keyword argument 'taba'
'''},
{'date': 'Thu Aug 06 2020 14:58:44.888 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 282
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 190
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 221
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 242
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module <module> line 53
    self.acessa(ocupante)
  module <module> line 71
    ocupante.ocupa(self)
  module <module> line 152
    self.vaga.sai()
AttributeError: 'Indio' object has no attribute 'vaga'
'''},
{'date': 'Thu Aug 06 2020 15:09:23.564 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 282
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 190
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 221
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 242
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module <module> line 53
    self.acessa(ocupante)
  module <module> line 71
    ocupante.ocupa(self)
  module <module> line 152
    self.vaga.sai()
AttributeError: 'Indio' object has no attribute 'vaga'
'''},
{'date': 'Thu Aug 06 2020 15:10:42.589 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 282
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 190
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 221
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 242
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module <module> line 53
    self.acessa(ocupante)
  module <module> line 71
    ocupante.ocupa(self)
  module <module> line 152
    self.vaga.sai()
AttributeError: 'Indio' object has no attribute 'vaga'
'''},
{'date': 'Thu Aug 06 2020 15:40:21.364 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 282
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 190
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 221
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 242
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module <module> line 53
    self.acessa(ocupante)
  module <module> line 71
    ocupante.ocupa(self)
  module <module> line 152
    self.vaga.sai()
AttributeError: 'Indio' object has no attribute 'vaga'
'''},
{'date': 'Thu Aug 06 2020 15:41:58.919 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 287
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 195
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 226
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 247
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module <module> line 53
    self.acessa(ocupante)
  module <module> line 71
    ocupante.ocupa(self)
  module <module> line 157
    self.vaga.sai()
AttributeError: 'method' object has no attribute 'sai'
'''},
{'date': 'Thu Aug 06 2020 15:42:44.130 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 287
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 195
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 226
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 247
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module <module> line 53
    self.acessa(ocupante)
  module <module> line 71
    ocupante.ocupa(self)
  module <module> line 159
    vaga.ocupou(self)
  module <module> line 84
    self.vazio.ocupa(ocupante)
AttributeError: 'Elemento' object has no attribute 'ocupa'
'''},
{'date': 'Thu Aug 06 2020 15:44:35.557 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 287
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 195
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 226
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 247
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module <module> line 53
    self.acessa(ocupante)
  module <module> line 71
    ocupante.ocupa(self)
  module <module> line 159
    vaga.ocupou(self)
  module <module> line 84
    self.vazio = ocupa(ocupante)
NameError: name 'ocupa' is not defined
'''},
{'date': 'Thu Aug 06 2020 15:45:59.453 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 287
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 195
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 226
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 247
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module <module> line 53
    self.acessa(ocupante)
  module <module> line 71
    ocupante.ocupa(self)
  module <module> line 159
    vaga.ocupou(self)
  module <module> line 91
    self.vazio.ocupa(ocupante)
AttributeError: 'Elemento' object has no attribute 'ocupa'
'''},
{'date': 'Thu Aug 06 2020 15:46:29.872 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 287
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 195
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 226
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 247
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module <module> line 53
    self.acessa(ocupante)
  module <module> line 71
    ocupante.ocupa(self)
  module <module> line 159
    vaga.ocupou(self)
  module <module> line 91
    self.vazio.ocupa(ocupante)
AttributeError: 'Elemento' object has no attribute 'ocupa'
'''},
{'date': 'Thu Aug 06 2020 15:49:35.87 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 294
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 202
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 233
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 277
    self.o_indio = Indio(imagem, x=0, y=0, cena=cena, taba=self)
TypeError: __init__() got an unexpected keyword argument 'taba'
'''},
{'date': 'Thu Aug 06 2020 16:18:03.451 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 282
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 190
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 221
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 242
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module <module> line 53
    self.acessa(ocupante)
  module <module> line 71
    ocupante.ocupa(self)
  module <module> line 152
    self.vaga.sai()
AttributeError: 'Indio' object has no attribute 'vaga'
'''},
{'date': 'Thu Aug 06 2020 16:18:27.806 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 282
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 190
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 221
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 242
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module <module> line 53
    self.acessa(ocupante)
  module <module> line 71
    ocupante.ocupa(self)
  module <module> line 152
    self.vaga.sai()
AttributeError: 'Indio' object has no attribute 'vaga'
'''},
{'date': 'Thu Aug 06 2020 16:23:43.58 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 284
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 190
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 221
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 242
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module <module> line 53
    self.acessa(ocupante)
  module <module> line 71
    ocupante.ocupa(self)
  module <module> line 152
    self.vaga.sai()
AttributeError: 'Indio' object has no attribute 'vaga'
'''},
{'date': 'Thu Aug 06 2020 16:24:11.610 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 284
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 190
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 221
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 265
    self.o_indio = Indio(imagem, x=0, y=0, cena=cena, taba=self)
TypeError: __init__() got an unexpected keyword argument 'taba'
'''},
{'date': 'Thu Aug 06 2020 17:25:27.577 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 287
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 195
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 227
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 245
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module <module> line 53
    self.acessa(ocupante)
  module <module> line 71
    ocupante.ocupa(self)
  module <module> line 156
    self.vaga.sai()
AttributeError: 'Indio' object has no attribute 'vaga'
'''},
{'date': 'Thu Aug 06 2020 17:28:31.888 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 287
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 195
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 227
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 245
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module <module> line 53
    self.acessa(ocupante)
  module <module> line 71
    ocupante.ocupa(self)
  module <module> line 156
    self.vaga.sai()
AttributeError: 'Indio' object has no attribute 'vaga'
'''},
{'date': 'Thu Aug 06 2020 17:29:16.478 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 287
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 195
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 227
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 245
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module <module> line 53
    self.acessa(ocupante)
  module <module> line 71
    ocupante.ocupa(self)
  module <module> line 156
    self.vaga.sai()
AttributeError: 'Indio' object has no attribute 'vaga'
'''},
{'date': 'Thu Aug 06 2020 17:30:46.804 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 287
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 195
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 227
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 245
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module <module> line 53
    self.acessa(ocupante)
  module <module> line 71
    ocupante.ocupa(self)
  module <module> line 156
    self.vaga.sai()
AttributeError: 'Indio' object has no attribute 'vaga'
'''},
{'date': 'Thu Aug 06 2020 17:31:29.156 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 287
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 195
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 227
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 245
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module <module> line 53
    self.acessa(ocupante)
  module <module> line 71
    ocupante.ocupa(self)
  module <module> line 156
    self.vaga.sai()
AttributeError: 'Indio' object has no attribute 'vaga'
'''},
{'date': 'Thu Aug 06 2020 17:36:21.689 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 287
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 195
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 227
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 245
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module <module> line 53
    self.acessa(ocupante)
  module <module> line 71
    ocupante.ocupa(self)
  module <module> line 156
    self.vaga.sai()
AttributeError: 'Indio' object has no attribute 'vaga'
'''},
{'date': 'Thu Aug 06 2020 17:37:48.814 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 289
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 195
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 227
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 247
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module <module> line 53
    self.acessa(ocupante)
  module <module> line 71
    ocupante.ocupa(self)
  module <module> line 156
    self.vaga.sai()
AttributeError: 'Indio' object has no attribute 'vaga'
'''},
{'date': 'Thu Aug 06 2020 17:54:11.246 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 287
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 195
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 227
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 245
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module <module> line 53
    self.acessa(ocupante)
  module <module> line 71
    ocupante.ocupa(self)
  module <module> line 156
    self.vaga.sai()
AttributeError: 'Indio' object has no attribute 'vaga'
'''},
{'date': 'Wed Aug 12 2020 17:22:20.868 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 70
  """Objeto tenta sair e secebe autorização para seguir"""
  ^
IndentationError: expected an indented block
'''},
{'date': 'Wed Aug 12 2020 17:22:37.487 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 238
  self.azimute = self.AZIMUTE[self.AZIMUTE.index(self.azimute)-3]
  ^
IndentationError: unexpected indent
'''},